### DISCUSSION TITLE
Failed to resolve custom block

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#834] [Q&A] [answered]
![Screenshot_2023-03-04-22-06-32-465_com sketchware remod](https://user-images.githubusercontent.com/37414496/222917916-c4541120-4770-4eb9-ad68-a424e20d382f.jpg)
/* Failed to resolve Custom Block's code: java.util.IllegalFormatFlagsException: Flags = ' ' */

**Comment by subhash16** [comment]
No answer

**Comment by j-px** [comment]
@subhash16 don't be this impatient and stop acting like a kid! We're doing this for free and will handle issues when we've got the time.

**Comment by subhash16** [comment]
Sir I am not acting like a kid. I appreciate yours hardwork,becoz of you we are getting latest version of sketchware pro. I thought that you haven't noticed the msg so I do that. I am sorry if I hurt you

**Comment by j-px** [comment]
There's a 2nd stray `%` after `%1$s ` that's causing the issue.

**Comment by subhash16** [comment]
No that % is the code itself that not a mistake

**Reply by j-px** [reply]
if you have to use the modulo operator, or in general literal `%` in Java's `String.format(String, Object...)`, you have to write escape it and write it as `%%`.

**Comment by j-px** [comment, marked as answer]
top7 how a reply can't be selected as answer  
anyway the answer is this:

> if you have to use the modulo operator, or in general literal `%` in Java's `String.format(String, Object...)`, you have to write escape it and write it as `%%`.
